src/core/utils/Debounce/Debounce.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { Debounce } from '../Debounce/Debounce';
+import { HtmlInput } from '../../../reset/HtmlInput/HtmlInput';
+
+describe('debounce', () => {
+  it('runs given function immediately when no debounce time is specified', () => {
+    const mockFunction = jest.fn(() => 12);
```

Comment:
It seems that it could be also like for the test to work:
`const mockFunction = jest.fn();`

Was there previously some more things to be done with that magic `12`?

Reply:
Ah, forgot that. it was used in an earlier version of the test but forgot to remove that part. Removed the magic 12 :)